### ISSUE TITLE
Goal shouldn't be overridable when configuring dashcard viz settings

### ISSUE THREAD

**Issue by anki-code** [metabase/metabase#4131]
It's just confuse me (FYI): you can set the "goal" number in Progress in two places:
1. in question card
2. in questions card settings on dashboard

I've forgot about this places and when I've changed the "goal" in questions card I've confused when see old number in dashboard.

**Comment by tlrobinson** [comment]
All of the chart settings are available on dash cards (except the "data" section) by design. I'm not sure it's undesirable to be able to change the goal on a dashboard. However, we should make it more clear that changing the settings only apply to the card on the dashboard.

**Comment by anki-code** [comment]
@mazameli I can't invent the case when I should use the Progress on dashboard with different goal number. Seems that Progress Goal has strong connection to context of the task of this progress.

In my case I have Progress for count of registered users and goal. When I change the goal in the question (for new month) I expect that all dashboards will be updated.

**Comment by mazameli** [comment]
Yeah, I agree. The weirdness lies with the fact of where we chose to put the input box to set the goal; I view the goal value of a progress bar chart not so much as a visualization setting and more part of the question's definition. (e.g., if I have a saved question called "How far away we are from our sale's goal this month," and then someone puts it on a dashboard but changes the goal, it's not the same card, really.) This will become more evident if and when we start allowing people to set goal values based on the results of other queries.

**Comment by ranquild** [comment]
This is a known limitation in viz settings. There is no way to remove viz settings from a dashboard card once set.

**Comment by perivamsi** [comment]
This is expected behavior. Closing the issue.
